src/SonarScanner.MSBuild.PreProcessor/SonarScanner.MSBuild.PreProcessor.csproj

```diff
@@ -13,6 +13,7 @@
     <ProjectReference Include="..\SonarScanner.MSBuild.Common\SonarScanner.MSBuild.Common.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.21.9" />
```

Comment:
We need something like this here to be sure that the CS file doesn't get out of date when we update the proto file:
```
  <!-- Generate protobuf classes -->
  <Target Name="GenerateProtobufClasses" BeforeTargets="CoreCompile">
    <Message Importance="high" Text="Generating protobuf classes..." />
    <Exec Command=""$(NugetPackageRoot)/Google.Protobuf.Tools/3.6.1/tools/windows_x64/protoc.exe" -I=./Protobuf --csharp_out=./Protobuf ./Protobuf/AnalysisCacheMsg.proto" />
    <Message Importance="high" Text="Protobuf classes generated." />
  </Target>
```
the original had also this, I don't think it's needed (didn't try)
```
  <Target Name="GenerateProtobufClasses" BeforeTargets="CoreCompile" Outputs="./Protobuf/AnalysisCacheMsg.cs" Inputs="./Protobuf/AnalysisCacheMsg.proto">
```

Reply:
(too complicated) source from sonar-dotnet:
https://github.com/SonarSource/sonar-dotnet/blob/638c7a6f4481677fc0c5c300d402d5d2aaeabad6/analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj#L56-L71

Reply:
Indeed, too complicated. I've installed `Grpc.Tools` and everything is handled under the hood by adding:
```
  <ItemGroup>
    <Protobuf Include="**/*.proto" />
  </ItemGroup>
```
Once it passes review and see that it's working fine, I will change also sonar-dotnet implementation.

Reply:
Sounds good. When on it, please update the license header there to `/ * ` comments too